### DISCUSSION TITLE
Screenpack "overlay" boxes

### DISCUSSION THREAD

**Discussion by potsmugen** [ikemen-engine/Ikemen-GO#1181] [Other] [answered]
Those dark boxes when you open a menu never seem to cover the whole screen when the game is set to widescreen. I think they should.

Screenshots:
![ikemen000](https://user-images.githubusercontent.com/107247004/233687038-bfc81085-9873-4cfa-8381-7e233bf69b6e.png)

![ikemen001](https://user-images.githubusercontent.com/107247004/233687045-e0b0d5b6-01cb-4c28-a1cd-ed853bcfcc09.png)

I'm not sure if this is intentional or not, so I'm posting it over here.

**Comment by Finally1994** [comment]
p1.window of [Victory Screen],bg.spr of [Face] in Fight.def, counter.anim & counter.spr of [Continue Screen] were also interrupted at this position and were not displayed.

**Comment by MUGEN-Slayer** [comment, marked as answer]
The features work as they should. If someone wants to use a 4:3 screenpack in a 16:9 resolution, the user just needs to modify the necessary parameters. Check this in infobox:

;overlay.window = 0,0, 320,240
overlay.col = 0,0,0
overlay.alpha = 210,0

Remove the semicolon and try something like this:
overlay.window =-54, 0, 380, 240

![ikemen000](https://user-images.githubusercontent.com/68881657/235548679-c38f797d-403f-4b71-9346-8e152178c12b.png)

The same goes for the pause menu.

**Reply by Finally1994** [reply]
![ikemen010](https://user-images.githubusercontent.com/127338553/235565227-f446630a-e1b9-4349-92a2-e35594f0fab5.png)
What about counter.anim and counter.spr in [Continue Screen]?
Black windows work fine, but anim and spr are choppy.

**Reply by MUGEN-Slayer** [reply]
It also works as it should. It's the same situation. If you want to use a 16:9 screenpack, then check the "data/mugen1" folder.

**Reply by potsmugen** [reply]
Ah. Yeah that makes perfect sense. The reason they didn't work with my (small) Mugen1 edit was that I made it 4:3.
But maybe, from a user's point of view, they could just default to covering the entire screen. At least with the dark boxes I don't see why someone would want it any other size.

**Reply by Finally1994** [reply]
It seems that the difference is whether or not localcoord is described in [Info]
Only cell.bg.spr, counter.anim and counter.spr are cut off and not displayed.
(Other BGs are displayed normally even if there is no description of localcoord)
